### DISCUSSION TITLE
How to relay outgoing emails through something like MailJet?

### DISCUSSION THREAD

**Discussion by cyl3x** [emersion/maddy#441] [Q&A] [answered]
Is it possible to route all outgoing emails through MailJet? If not, how would i redirect the traffic to a smtp relay?

**Comment by foxcpp** [comment, marked as answer]
MailJet offers an SMTP relay service: https://www.mailjet.com/feature/smtp-relay/

To use it for all outbound traffic, replace target.remote in maddy.conf with target.smtp and [configure it appropriately](https://github.com/foxcpp/maddy/blob/dev/docs/reference/targets/smtp.md#configuration-directives).
Example:
```
target.smtp outbound_delivery {
  targets tcp://in-v3.mailjet.com:25
  require_tls 
  auth plain USERNAME PASSWORD
}
```
